tests/cli_autogen_input/autogen_test/sonic-device_metadata.yang

```diff
@@ -0,0 +1,123 @@
+module sonic-device_metadata {
```

Comment:
Two questions:

1) Why are we using a special Yang for test? Why not use YANG model in src/sonic-yang-models?
2) Can we extend the unit test to use every YANG model and the test code to test the generated CLI with expected behavior in data per YANG?

Reply:
Answer to question 2:
For the `sonic-cli-gen` utility there are only 2 types of YANG models - one that has a `list` entity and the one that does not have a `list` entity.  That is why in `tests/cli_autogen_test.py` we have test cases only for 2 YANG models.